### PR TITLE
Next

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: app/screens/reader-screen/reader-screen.tsx [hunk 1]
```diff
@@ -52,72 +64,119 @@ export class ReaderScreen extends React.Component<ReaderScreenProps, {}> {
...
 
-  private renderContent = (content: Content) => (
-    <ContentListItem
-      key={content.url}
-      content={content}
-      onPressItem={this.onPressContentItem}
-    />
-  )
+  private onLoadMore = () => {
+    console.tron.log("ONLOADMORE")
```
Comment (nnkken) [inline app/screens/reader-screen/reader-screen.tsx]: for debug?

Comment (nwingt) [reply app/screens/reader-screen/reader-screen.tsx]: Removed

File: app/models/root-store/setup-root-store.ts [hunk 2]
```diff
@@ -94,10 +96,25 @@ export async function setupRootStore() {
...
+      },
       ...snapshot
       /* eslint-enable @typescript-eslint/no-unused-vars */
     }) => storage.save(ROOT_STATE_STORAGE_KEY, {
       ...snapshot,
+      readerStore: {
+        contents: Object
+          .values(contents)
+          .slice(0, 1000) // Cache 1,000 contents at max
+          .sort(sortContentForSnapshot)
```
Comment (nnkken) [inline app/models/root-store/setup-root-store.ts]: should slice after sort?

Comment (nwingt) [reply app/models/root-store/setup-root-store.ts]: Fixed

**Pull request merged**